### ISSUE TITLE
Syntax highlighting

### ISSUE THREAD

**Issue by flip111** [krakjoe/phpdbg#140]
Would it be possible to add syntax highlighting when doing `list 30`?

**Comment by bwoebi** [comment]
Possible, definitely… doing it nicely… not that easy.

Problem is that you have no idea what terminal color somebody is using. If someone has a white background other colors are looking nice than when the terminal is black...

**Comment by flip111** [comment]
@bwoebi isn't it a reponsiblity of the user to set sane terminal colors? Perhaps the user can even change colors before starting phpdbg. Anyway if the option is there it would be good (possible to be turned off with a switch)

**Comment by Llbe** [comment]
I agree syntax highlighting would be a good feature. But it also leads to other things: ability to adjust the colors and patterns, to search among the listed code etc.

Maybe an ability to pipe the output to $EDITOR would be the best approach?
